docs/document/content/features/dist-sql/syntax/ral/ral.cn.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "RAL"
+weight = 1
++++
+
+## 定义
+
+RAL (Resource & Rule Administrate Language) 为 Apache ShardingSphere 的管理语言，负责Hint、事务类型切换、分片执行计划查询等增量功能的操作。
+
+## 使用实战
+
+| 语句                                                | 说明                                                           | 示例                                           |
+|:---------------------------------------------------|:--------------------------------------------------------------|:-----------------------------------------------|
+|set variable transaction_type = xx                  | 修改当前连接的事务类型, 支持LOCAL，XA，BASE                         | set variable transaction_type = XA            |  
```

Comment:
Do we have the default value?

Reply:
Default value is "LOCAL":
PROXY_TRANSACTION_TYPE("proxy-transaction-type", "LOCAL", String.class),